File: snippets/typescript/query_batch_points.ts
import { QdrantClient } from "@qdrant/js-client-rest";

const client = new QdrantClient({ host: "localhost", port: 6333 });

client.queryBatch("{collection_name", {
    searches: [{
        query: [0.01, 0.45, 0.67]
    },
    {
        query: [0.11, 0.35, 0.6]
    }]
});

File: snippets/go/query_batch_points.go
package client

import (
	"context"

	"github.com/qdrant/go-client/qdrant"
)

func queryBatch() {
	client, err := qdrant.NewClient(&qdrant.Config{
		Host: "localhost",
		Port: 6334,
	})
	if err != nil {
		panic(err)
	}

	client.QueryBatch(context.Background(), &qdrant.QueryBatchPoints{
		CollectionName: "{collection_name}",
		QueryPoints: []*qdrant.QueryPoints{
			{
				CollectionName: "{collection_name}",
				Query:          qdrant.NewQuery(0.01, 0.45, 0.67),
			},
			{
				CollectionName: "{collection_name}",
				Query:          qdrant.NewQuery(0.11, 0.35, 0.6),
			},
		},
	})
}
